workflow_tests/utils/preGenerateTest.ts

```diff
@@ -87,7 +81,7 @@ describe('test workflow ${workflowName}', () => {
         };
         const result = await act
             .runEvent('[EVENT]', {
-                workflowFile: path.join(repoPath, '.github', 'workflows'),
+                workflowFile: path.join(repoPath, '.github', 'workflows', '${workflowName}.yml'),
```

Comment:
Why this change?

Reply:
Because otherwise, it was generating a template that wouldn't work from the beginning. This script has been created as a helper for creating new workflow templates, but wasn't well maintained, I've adjusted this to the typescript, with a couple of improvements.